pos_payment_method_cashdro/models/pos_payment_method.py

```diff
@@ -13,10 +13,13 @@ def _get_payment_terminal_selection(self):
         string="Cashdro Terminal Host Name or IP address",
         help="It must be reachable by the PoS in the store",
     )
-    cashdro_user = fields.Char(string="Cashdro User")
-    cashdro_password = fields.Char(string="Cashdro Password")
+    cashdro_user = fields.Char()
+    cashdro_password = fields.Char()
 
-    def _onchange_is_cash_count(self):
-        if self.use_payment_terminal == "cashdro":
-            return
-        return super()._onchange_is_cash_count()
+    def _onchange_journal_id(self):
+        """Cash payment method force the `use_payment_terminal` to `False` as
+        it's assumed that a cash journal can't have a payment terminal"""
+        if self.use_payment_terminal != "cashdro":
+            return super()._onchange_journal_id()
+        self.use_payment_terminal = "cashdro"
```

Comment:
Shouldn't this be done only for cash journals? Or let decide the user and do nothing

Reply:
I just improved the usability. This is done to avoid the change of the journal drop the terminal config.